keyboards/sirius/unigo66/keymaps/danielhklein/keymap.c

```diff
@@ -0,0 +1,189 @@
+  XXXXXXX, XXXXXXX,_______,                                                                             _______, XXXXXXX, XXXXXXX,
+                                                XXXXXXX,   _______, XXXXXXX, XXXXXXX,
+                                                XXXXXXX,   _______, XXXXXXX, XXXXXXX
+    ),
+
+/* ADJ
+ *
+ * ,--------------------------------------------------.           ,--------------------------------------------------.
+ * |   F1   |  F2  |  F3  |  F4  |  F5  |  F6  |      |           |Nlock |  =   |  /   |  *   |  -   |      |        |
+ * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
+ * |   F7   |  F8  |  F9  |  F10 |  F11 |  F12 |      |           |      |  7   |  8   |  9   |  +   |      |        |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |        |      |      |      |      |      |------|           |------|  4   |  5   |  6   | Enter|      |        |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |        |      |      |      |      |      |      |           |      |  1   |  2   |  3   | Space|      |        |
+ * `--------+------+------+---------------------------'           `---------------------------+------+------+--------'
+ *   |      |      |      |                                                                   |  0   |  .   |      |
+ *   `--------------------'                                                                   `--------------------'
+ *                                        ,-------------.       ,--------------.
+ *                                        |      |      |       |      |       |  
+ *                                        |      |      |       |      |       | 
+ *                                        |      |      |       |      |       | 
+ *                                        `-------------'       `--------------'
+ */
+    [_ADJ] = LAYOUT(
+  KC_F1,   KC_F2,  KC_F3,   KC_F4,   KC_F5,   KC_F6,   XXXXXXX, KC_NLCK, KC_PEQL, KC_PSLS, KC_PAST, KC_PMNS, XXXXXXX, XXXXXXX,
+  KC_F7,   KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, XXXXXXX, KC_P7,   KC_P8,   KC_P9,   KC_PPLS, XXXXXXX, XXXXXXX,
+  XXXXXXX, XXXXXXX,XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   KC_P4,   KC_P5,   KC_P6,   KC_PENT, XXXXXXX, XXXXXXX, 
+  XXXXXXX, XXXXXXX,XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_P1,   KC_P2,   KC_P3,   KC_SPC,  XXXXXXX, XXXXXXX,
+  XXXXXXX, XXXXXXX,XXXXXXX,                                                                         KC_P0,   KC_PDOT, XXXXXXX,
+                                              XXXXXXX, XXXXXXX, _______, XXXXXXX,
+                                              XXXXXXX, XXXXXXX, _______, XXXXXXX
+    )
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Lines 133-142 aren't needed at all and should be deleted. Both functions are deprecated.